airflow/utils/types.py

```diff
@@ -71,3 +71,10 @@ class EdgeInfoType(TypedDict):
     """Extra metadata that the DAG can store about an edge, usually generated from an EdgeModifier."""
 
     label: str | None
+
+
+@runtime_checkable
+class Stringable(Protocol):
+    def __str__(self) -> str:
+        pass
```

Comment:
🤔 Is this required? Special method `__str__` should be implemented (implicit or explicit) in additional should return string, otherwise TypeError will raised

Reply:
I wanted to make it explicit that Stringable must have a __str__. 
🤔I know this is not cool but I don't have idea about this.